src/main/java/mate/academy/springbootwebgreqit/dto/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springbootwebgreqit.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+public class UpdateBookRequestDto {
```

Comment:
What the difference in fields between UpdateBookRequestDto and CreateBookRequestDto?